archive/app/controllers/ArchiveController.scala

```diff
@@ -31,16 +39,45 @@ class ArchiveController(redirects: RedirectService, renderer: Renderer, val cont
 
   private val redirectHttpStatus = HttpStatus.SC_MOVED_PERMANENTLY
 
-  def lookup(path: String): Action[AnyContent] = Action.async{ implicit request =>
+  def remoteRender404: Future[String] = ws.url(Configuration.moon.moonEndpoint)
+      .withRequestTimeout(6000.millis)
```

Comment:
6 seconds feels a bit generous here. We have a 2 second timeout architectural rule in Dotcom across the whole stack, so that is the upper bound really.